qiskit/transpiler/preset_passmanagers/level3.py

```diff
@@ -193,6 +194,7 @@ def _opt_control(property_set):
     init.append(RemoveResetInZeroState())
     init.append(OptimizeSwapBeforeMeasure())
     init.append(RemoveDiagonalGatesBeforeMeasure())
+    init.append(ElideSwaps())
```

Comment:
I think this will obsolete `OptimizeSwapBeforeMeasure`, and we likely want to move `ElideSwaps` to before `RemoveDiagonalGatesBeforeMeasure`, because `swap` isn't a diagonal gate, and their elision might give more opportunities for removal.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/9523/commits/438e0a98ec6a738825918d57c4ba3a1ab2c02289